Python/compile.c

```diff
@@ -883,13 +889,25 @@ compiler_use_next_block(struct compiler *c, basicblock *block)
 }
 
 static basicblock *
-compiler_copy_block(struct compiler *c, basicblock *block)
+new_basicblock_after(basicblock *prev)
```

Comment:
From the name, I would expect `prev->b_next = result` rather than `prev->b_list = result`.
I don't have a better name to suggest, but maybe add a comment.